PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,2 @@
 * [ ] Change is tested locally
-* [ ] Demo is updated to exercise change (if applicable)
```

Comment:
Can we keep this and introduce a demo later?

Reply:
I'm fine with that.

Addressed in 223580e